docs/browser-compatibility.md

```diff
@@ -0,0 +1,10 @@
+# Browser compatibility
+
+The Reaction Commerce component library aims to support all browsers supported by React v16, outlined [here](https://reactjs.org/docs/react-dom.html#browser-support).
```

Comment:
Since we no longer follow the React list, maybe something like:
> The Reaction Commerce component library works to support all modern browsers and IE11. 

Maybe some caveat about supporting IE11 in major functionality only and not in pixel-perfect representation?